### DISCUSSION TITLE
Prisma model: array of array of Int

### DISCUSSION THREAD

**Discussion by jambudipa** [prisma/prisma#17663] [Q&A] [answered]
I have a field in my model that needs to be an array of array of `Int`. I first tried this:
```
model TradablePair {
  // ...
  leverageBuy       Int[][]
  // ...
}
```
which obviously didn't work. In mitigation, I saw somewhere someone doing this:
```
model TradablePair {
  // ...
  leverageBuy       IntArr[]
  // ...
}

type IntArr {
  array Int[]
}
```
which seems like a reasonable workaround for the moment, but `prisma.tradablePair.create()` falls over with that. I don't know how I'm supposed to map an 2D array of `[[0,1],[2,3]]` to fit that format.

Can anyone help?

**Comment by jambudipa** [comment, marked as answer]
Ok, I realised on pressing "Send":
```
[{array: [0,1]},{array: [2,3]}]
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
